runtime/doc/lsp.txt

```diff
@@ -700,10 +701,39 @@ Lua module: vim.lsp                                                 *lsp-core*
       • {filetypes}?     (`string[]`) Filetypes the client will attach to, if
                          activated by `vim.lsp.enable()`. If not provided,
                          then the client will attach to all filetypes.
-      • {root_markers}?  (`string[]`) Directory markers (.e.g. '.git/') where
-                         the LSP server will base its workspaceFolders,
-                         rootUri, and rootPath on initialization. Unused if
-                         `root_dir` is provided.
+      • {root_markers}?  (`(string|string[])[]`) Directory markers (.e.g.
+                         '.git/') where the LSP server will base its
+                         workspaceFolders, rootUri, and rootPath on
+                         initialization. Unused if `root_dir` is provided.
+
+                         It's possibile to specify marker's priority. The
```

Comment:
This is misleading, because priority/precedence is always in effect. What we actually want to say is that you can opt-in to "equal precedence".
```suggestion
                         The list order decides the priority. To indicate "equal priority", specify names in a nested list.
```

Reply:
My bad, I re-generated the docs without updating the lua file.